tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/IoDataGenerationTest.java

```diff
@@ -275,4 +290,78 @@ else if (e.label().equals("writtenBy"))
         GraphSONWriter.build().mapper(GraphSONMapper.build().embedTypes(true).create()).create().writeGraph(os4, g);
         os4.close();
     }
+
+    @Test
+    public void shouldWriteGratefulDeadGraphSONV2d0() throws IOException {
+        final TinkerGraph g = TinkerGraph.open();
+        final TinkerGraph readG = TinkerGraph.open();
+
+        final GraphReader reader = GryoReader.build().create();
+        try (final InputStream stream = AbstractGremlinTest.class.getResourceAsStream("/org/apache/tinkerpop/gremlin/structure/io/gryo/grateful-dead.kryo")) {
+            reader.readGraph(stream, g);
+        }
+        final OutputStream os2 = new FileOutputStream(tempPath + "grateful-dead-V2d0-typed.json");
+        GraphSONWriter.build().mapper(GraphSONMapper.build().version(GraphSONVersion.V2_0).typeInfo(GraphSONMapper.TypeInfo.PARTIAL_TYPES).create()).create().writeGraph(os2, g);
+        os2.close();
+
+        final InputStream is = new FileInputStream(tempPath + "grateful-dead-V2d0-typed.json");
+        GraphSONReader.build().mapper(GraphSONMapper.build().version(GraphSONVersion.V2_0).typeInfo(GraphSONMapper.TypeInfo.PARTIAL_TYPES).create()).create().readGraph(is, readG);
+        is.close();
+
+        assertEquals(approximateGraphsCheck(g, readG), true);
+    }
+
+    /**
+     * Checks sequentially vertices and egdes of both graphs. Will check sequentially Vertex IDs, Vertex Properties IDs
+     * and values and classes. Then same for edges. To use when serializing a Graph and deserializing the supposedly
+     * same Graph.
+     */
+    private boolean approximateGraphsCheck(Graph g1, Graph g2) {
```

Comment:
So, since now the build creates the new v2d0 graphs, and it seems like the other ones are pushed in the repo, should I push the new ones too? Also, it seems that the already pushed ones have modifications. Should I push all of as well ? Here's the diff : https://gist.github.com/newkek/4e0488268f9bb78e0d3597821ae6b357